### DISCUSSION TITLE
Store response (-sr) ignore protocol and store only one response

### DISCUSSION THREAD

**Discussion by MuhammadUsman-coder** [projectdiscovery/httpx#800] [Q&A] [answered]
There is a bug  in httpx . While using -sr flag to store responses. 
Supposed we got below two URLs with different applications running on http and https.

http://example.com
https://example.com

When i run 
`cat urls.txt|httpx -sr`

It will only create one file in the output folder with name example.com. I think the file name should also include http or https in the start and store response for both URLs.

**Comment by ehsandeep** [comment, marked as answer]
@MuhammadUsman-coder, httpx is default to https and fallback to http when https is not accessible, if you want to probe/store both ports, you can use `-no-fallback` option.

```
cat urls.txt | httpx -sr -nf
```

**Reply by MuhammadUsman-coder** [reply]
Thank you
